chcemvediet/locale/sk/LC_MESSAGES/django.po

```diff
@@ -1347,6 +1347,12 @@ msgstr "Chcem vedieť viac"
 msgid "main:homepage:signup:button"
 msgstr "Registrácia"
 
+msgid "main:homepage:casestudies:anchor"
+msgstr "vybrane-pripady"
+
+msgid "main:homepage:casestudies:heading"
+msgstr "Vybrané prípady"
```

Comment:
Prekladove stringy sme zvykli mat usporiadane podla abecedy. Cize `main:homepage:c...*` by malo ist az za `main:homepage:b...`

Ale pozeram, ze to nemame usporiadane. Asi by sa zislo to v separatnom PR usporiadat.

Reply:
prekladovi string som vytvaral cez `makemessage`. https://github.com/martinmacko47/chcemvediet/issues/270 . Mam teda zmenit poradie aj napriek tomu ako ho makemassage zaradilo?

Reply:
Hm. Bol som v tom, ze `makemessage` to pridava v abecednom poradi. Ano, pls nechaj poradie ake dava `makemessage`. Ak sa ti chce, skus zistit/pogooglit, ze podla coho to vlastne `makemessage` sorti, nech vieme.

Reply:
:ok: